source/includes/connect/atlas.php

```diff
@@ -14,6 +14,5 @@
 $result = $admin->command($command)->toArray();
 
 echo json_encode($result), "\n";
-echo "Pinged your deployment. You successfully connected to MongoDB!\n";
+echo 'Pinged your deployment. You successfully connected to MongoDB!\n';
```

Comment:
S: should we replace all \n chars with `, PHP_EOL`?

Reply:
sure I can do that